pkg/apis/componentconfig/edgesite/v1alpha1/default.go

```diff
@@ -31,10 +32,7 @@ import (
 
 // NewDefaultEdgeSiteConfig returns a full EdgeSiteConfig object
 func NewDefaultEdgeSiteConfig() *EdgeSiteConfig {
-	hostnameOverride, err := os.Hostname()
-	if err != nil {
-		hostnameOverride = constants.DefaultHostnameOverride
-	}
```

Comment:
There are many places that get the `hostnameOverride`, could we move `func getHostname() string {` to a common place, then import it in all places? :) And please squash the commits.

Reply:
well done~